### ISSUE TITLE
Fix PyPi README

### ISSUE THREAD

**Issue by thinkingserious** [sendgrid/open-source-library-data-collector#14]
https://pypi.python.org/pypi/open_source_library_data_collector/

**Comment by andela-ijubril** [comment]
Hi @thinkingserious can you expatiate on this?

**Comment by thinkingserious** [comment]
Hi @andela-ijubril,

The README is not showing up on PyPi. It should look something like this: https://pypi.python.org/pypi/sendgrid

**Comment by andela-ijubril** [comment]
Hi @thinkingserious I just sent a PR for this issue and I saw that if failed to pass on Travis.
While setting up on my machine I also discovered that it is breaking. I think you might want to include the version of each of the package you have that is working at your end in the requirements file, because when I try to downgrade some packages I passed some blockers while setting up `e.g I downgraded sendgrid to sendgrid==2.2.1` and I met another blocker in the github package. so I feel explicitly specifying the packages with version numbers would help pass the blocker.
Also, I sent in the contributors agreement already.
Thanks... I'm sorry for the long comment
